lib/twiglet/error_serialiser.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Twiglet
+  class ErrorSerialiser
+    def serialise_error(error)
+      error_fields = {
+        error: {
+          type: error.class.to_s,
+          message: error.message
+        }
+      }
+      add_stack_trace(error_fields, error)
```

Comment:
As much as I prefer "Stack Trace", Ruby calls it Backtrace, so would be more intuitive for a Ruby dev to see it named as such :)

Reply:
i dont mind, i was just keeping existing interface, tokenhouse was logging it as stack trace right now
you sure you wanna change that? @moltenice

Reply:
Oh in that case I'll leave it up to you!